pyanaconda/payload/livepayload.py

```diff
@@ -98,9 +98,9 @@ def unsetup(self):
         except OSError:
             pass
 
```

Comment:
I think it would make sense to log a warning if there are packages/groups requested to be added but the current payload is not package based/does not support adding additional packages.

I'm not sure where the best place for this is though - I guess either when attempts are made to add the packages/groups or when the payload API methods that would normally install the packages/groups are executed.